thread_pthread.h

```diff
@@ -59,6 +59,7 @@ typedef struct rb_global_vm_lock_struct {
      *   timer.
      */
     struct ccan_list_head waitq; /* <=> native_thread_data_t.node.ubf */
+    rb_atomic_t waiting;
```

Comment:
ah, you are adding a new counter!!
Honestly speaking, I don't want to add `waiting` counter, especially you use ATOMIC instructions.
It is expensive operation.

Reply:
I think this counter could be removed and implemented in the extension.